nni/experiment/experiment.py

```diff
@@ -193,8 +207,369 @@ def run(self, port: int = 8080, debug: bool = False) -> bool:
             self.stop()
 
 
-    def get_status(self) -> str:
+    def connect_experiment(self, port: int):
```

Comment:
it would be better to have a BaseExperiment class which has basic functionality of an experiment, then write different Experiment classes inheriting BaseExperiment to provide view/launch experiment and to manage experiments

Reply:
good point, we need to discuss this, in the previous discussion, we use parameters to distinguish different launch ways.